Server/src/lib/envHelper.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+const appSettings = require('../../appsettings.json');
+
+export const getResourceConnectionString = (): string => {
+  const resourceConnectionString = process.env['ResourceConnectionString'] || appSettings.ResourceConnectionString;
+
+  if (!resourceConnectionString) {
+    throw new Error('No ACS connection string provided');
+  }
+
+  return resourceConnectionString;
+};
+
+export const getEnvUrl = (): string => {
+  const uri = new URL(getResourceConnectionString().replace('endpoint=', ''));
```

Comment:
Was this a hack we did or is this expected way to get the endpoint URL? (Should we instead require endpoint url be set in the appsettings.json?)

Reply:
Created an ADO item
https://skype.visualstudio.com/SPOOL/_workitems/edit/2691366